lib/cucumber/util/colors.js

```diff
@@ -1,15 +1,15 @@
-var colors = require('colors/safe');
+function Colors (useColors) {
+  var colors = require('colors/safe');
```

Comment:
Any reason to keep this within the function and not at the top of the file in a more conventional way?